language/move-prover/tests/sources/functional/emits.move

```diff
@@ -0,0 +1,14 @@
+module EmitsTest {
```

Comment:
Can you add some negative tests as well?

And actually those tests belong into the move-model/tests directory. Here you would later test whether the connection to the boogie backend works. But before you should just test the language level in the above place.

Reply:
I moved this test case to "move-model/tests/sources", and added a negative test there as well.